loading.bs

```diff
@@ -331,6 +371,304 @@ Note: Applying the signed exchange's response here has the effect of letting a
+    Link: <https://cdn.publisher.example/lib.js>;
+        rel="allowed-alt-sxg";
+        header-integrity="sha256-XXXXXX"
+
+    <html>
+      <head>
+        <script src="https://cdn.publisher.example/lib.js"></script>
+      </head>
+      ...
+    </html>
+```
+
+If another page includes a `<link rel="prefetch"
+href="https://feed.example/sxg.publisher.example/page.html.wbn">`, the above
+response allows it to transitively prefetch a signed exchange that it can use to
+satisfy the `https://cdn.publisher.example/lib.js` `<script>` element.
+
+</div>
+
+* The <{Link}> element.
+
+    The <dfn element-attr for="link">variants</dfn> parameter may be present.
+
+    The <dfn element-attr for="link">variant-key</dfn> parameter may be
+        present.
+
+* Link type "<dfn attr-value for="link/rel">allowed-alt-sxg</dfn>"
+
+    The <{link/rel/allowed-alt-sxg}> keyword may be used in the HTTP <a
+    http-header>Link</a> header of the inner HTTP response of the main resource
+    signed exchange to indicate that the content of the [=Link Target=] is also
+    available as a particular signed exchange identified by the
+    <{link/header-integrity}> parameter. The <dfn element-attr
+    for="link">header-integrity</dfn> parameter holds the value of the
+    [=response/header integrity value=] of the [=alternate signed exchange=],
+    encoded as a [[CSP]] <a grammar>hash-source</a>. 
+* An <dfn>alternate signed exchange link</dfn> is a `` `Link` `` header sent
+    with a signed exchange |S|, with the <{link/rel/alternate}> link type, the
+    `type` parameter set to `application/signed-exchange`, and a [=Link
+    Context=] (an `anchor` parameter). The [=Link Context=] MUST be the [=Link
+    Target=] of a <{link/rel/preload}> `` `Link` `` header inside the signed
+    content of |S|, and the [=alternate signed exchange link=] means that this
+    resource can also be found inside the <dfn dfn-type=dfn>alternate signed
+    exchange</dfn> at the [=Link Target=] of the [=alternate signed exchange
+    link=].
+
+
+The [=appropriate time=] to [=fetch and process the linked resource=] for an
+[=alternate signed exchange link=] is when the following steps return true:
+
+<ol algorithm="appropriate time for an alternate signed exchange link">
+
+1. Let |R| be the [=response=] created from the signed exchange in the response
+    that held the [=alternate signed exchange link=].
+1. Let |L| be the set of <{link/rel/allowed-alt-sxg}> <a http-header>Link</a>
+    HTTP headers on |R| whose [=Link Target=] is the same as the [=alternate
+    signed exchange link=]'s [=Link Context=] and whose <{link/variants}>
+    parameter and <{link/variant-key}> parameter are the same as the [=alternate
+    signed exchange link=]'s parameters if they are present.
+1. If |L| is empty, return false.
+1. Let |preload links| be the set of <{link/rel/allowed-alt-sxg}> <a
+    http-header>Link</a> HTTP headers on |R|.
+1. For each |preload link| in |preload links|:
```

Comment:
I would probably use this line to explain why we create link elements and requests that seem unused.

```suggestion
1. For each |preload link| in |preload links|, use the following steps to resolve the link's actual target, incorporating attributes like {<link/imagesrcset>} and <{link/imagesizes}>}:
```